mass_editing/i18n/am.po

```diff
@@ -0,0 +1,187 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * mass_editing
+# 
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2016
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 9.0c\n"
```

Comment:
Still has the old translation?

Reply:
I think it's because headers are not updated unless you export from Odoo.